companies/oculuscom.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "oculuscom",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "entertainment",
+        "ads",
+        "social media"
+    ],
+    "name": "Facebook Technologies Ireland Limited",
+    "runs": [
+        "Oculus"
+    ],
+    "address": "Facebook Technologies Ireland Limited ATTN:\nOculus Privacy Operations\n4 Grand Canal Square, Grand Canal Harbour\nDublin 2, Ireland",
+    "email": "DPO-Inquiries@oculus.com",
+    "web": "https://www.oculus.com",
+    "sources": [
+        "https://www.oculus.com/legal/privacy-policy/",
+        "https://www.oculus.com/legal/about-us/",
+        "https://github.com/datenanfragen/data/issues/609"
+    ],
+    "comments": [
+        "ehemals Oculus VR Ireland Limited"
```

Comment:
We add former company names to the `runs` list, adding a suffix. In German, I always use `(ehem.)`, in English `(formerly)` seems appropriate.

As this record is relevant for all countries, we unfortunately need to go with English.